scripts/gen/windows-resources

```diff
@@ -15,30 +16,28 @@ trap 'rm -rf $TEMPDIR' EXIT
 
 if [ "$(go env GOHOSTOS)" == "windows" ]; then
 	WINDRES=windres
-	WINDMC=windmc
 else
 	# Cross compiling
 	WINDRES=x86_64-w64-mingw32-windres
-	WINDMC=x86_64-w64-mingw32-windmc
 fi
 
 # Generate a Windows file version of the form major,minor,patch,build (with any part optional)
-VERSION_QUAD=$(echo -n $VERSION | sed -re 's/^([0-9.]*).*$/\1/' | tr . ,)
+VERSION_QUAD=$(echo -n "$VERSION" | sed -re 's/^([0-9.]*).*$/\1/' | tr . ,)
 
 # Pass version and commit information into the resource compiler
 defs=
-[ ! -z $VERSION ]      && defs="$defs -D DOCKER_VERSION=\"$VERSION\""
-[ ! -z $VERSION_QUAD ] && defs="$defs -D DOCKER_VERSION_QUAD=$VERSION_QUAD"
-[ ! -z $GITCOMMIT ]    && defs="$defs -D DOCKER_COMMIT=\"$GITCOMMIT\""
+[ ! -z "$VERSION" ]      && defs="$defs -D DOCKER_VERSION=\"$VERSION\""
+[ ! -z "$VERSION_QUAD" ] && defs="$defs -D DOCKER_VERSION_QUAD=$VERSION_QUAD"
+[ ! -z "$GITCOMMIT" ]    && defs="$defs -D DOCKER_COMMIT=\"$GITCOMMIT\""
 
 function makeres {
 	$WINDRES \
-		-i $RESOURCES/$1 \
-		-o $3 \
-		-F $2 \
+		-i "$RESOURCES/$1" \
+		-o "$3" \
+		-F "$2" \
 		--use-temp-file \
-		-I $TEMPDIR \
-		$defs
+		-I "$TEMPDIR" \
+		"$defs"
```

Comment:
I think the build failures are related to this line?

I guess `defs` should be an array instead of a string?